frontend/containers/image-gallery/component.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC, useState } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import Image from 'next/image';
+
+import Button from 'components/button';
+import Carousel, { Slide } from 'components/carousel';
+import Modal from 'components/modal';
+
+import { ImageGalleryProps } from '.';
+
+export const ImageGallery: FC<ImageGalleryProps> = ({ images }) => {
+  // Sets the index of the image displayed on carousel. The default is undefined for not displaing any image.
+  const [active, setActive] = useState<number>();
+  const { formatMessage } = useIntl();
+
+  return (
+    <div area-hidden>
+      <ul className="flex gap-1">
+        {images.map(({ file: { small } }, index) => {
+          const selected = active === index;
+          return (
+            <li key={small} className="inline">
+              <Button
+                theme={selected ? 'secondary-green' : 'naked'}
+                className={cx('block w-8 h-8 px-0 py-0 rounded-none  transition-all', {
```

Comment:
The buttons are missing their rounded outline when navigating via keyboard (to match all others when navigating via keyboard). 

<img width="222" alt="Screenshot 2022-05-26 at 11 12 55" src="https://user-images.githubusercontent.com/6273795/170470172-fb316c67-26fb-495d-a625-853cf19cd404.png">

Reply:
I added the outline. It is not so rounded because the images are squares.

Reply:
Weird, it's not showing for me. It may be just my browser acting up.

Reply:
Sorry the preview link was old. Try this one: https://heco-invest-frontend-ipghs4ads-vizzuality1.vercel.app/en/project/test-7-test